src/release/process.md

```diff
@@ -50,6 +59,14 @@ docker exec -d -it rcs bash -c 'promote-release /tmp/stable stable /data/secrets
 
 ## Promote master to beta (T-2 days, Tuesday)
 
+We need to find out the parent commit in which the PR opened last Monday merged.
+
+Go to that PR, and find the "bors merged commit $SHA into rust-lang:master at the bottom.
+
+Locally, run `git log $SHA --author=bors`, and take the SHA of the first
+bors-authored merge into master (should be the the first commit). Call that
+`$BRANCH_POINT`.
```

Comment:
You can get the first parent of a merge commit with `git rev-parse $SHA^`.

Reply:
Ah, right. I had written this before changing to rollup=never above, will update.